string uav_id
int32 uav_priority
int64 status_code
string status_name
geometry_msgs/Point current_destination

